app/views/dashboard/show.html.erb

```diff
@@ -7,14 +7,13 @@
         </h1>
         <div class="index-group">
           <h2 class="govuk-heading-m">
-<!--            TODO Fix after Ethan code merge for lead providers-->
-            <%= govuk_link_to "Find and add schools", nil %>
+            <%= govuk_link_to "Find and add schools", start_lead_providers_report_schools_path %>
           </h2>
           <p class="govuk-body">Search for schools you want to recruit and add schools you have partnered with.</p>
         </div>
         <div class="index-group">
           <h2 class="govuk-heading-m">
-            <%= govuk_link_to "Check your schools for #{@lead_provider.cohorts.find_by(start_year: Time.zone.now.year).start_year}", lead_providers_your_schools_path(selected_cohort_id: @lead_provider.cohorts.find_by(start_year: Time.zone.now.year).id) %>
+            <%= govuk_link_to "Check your schools for #{@lead_provider.cohorts.find_by(start_year: Time.zone.now.year)&.start_year}", lead_providers_your_schools_path(selected_cohort_id: @lead_provider.cohorts.find_by(start_year: Time.zone.now.year)&.id) %>
```

Comment:
Would `@lead_provider.cohorts.current.&id` work here as well?

Reply:
Ehh, the cohort would be missing if we messed up seeding. I will look into making it nicer.